export function resetErrors(setError) {
  setError({
    error_triggered: false
  })
}
